content/java/concepts/comparable/comparable.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'Comparable'
+Description: 'The Comparable interface is used to define how a class is to be sorted.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Collections'
+  - 'Comparison'
+  - 'Data Structures'
+  - 'Sort'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Comparable`** interface is used to define how a [class](https://www.codecademy.com/resources/docs/java/classes) is to be sorted. It is not to be confused with the [`Comparator`](https://www.codecademy.com/resources/docs/java/comparator) interface, which is implemented in a separate class. The `Comparable` interface is implemented in the class to be sorted.
+
+## Syntax
+
+```pseudo
+class MyClass implements Comparable<MyClass> {
+  // Class body.
+  ...
+  @Override public int compareTo(MyClass value)
+  {
+      // Comparison Logic
+      ...
+      return result;
+  }
+}
+```
+
+Applying the `Comparable` interface to `MyClass` requires the `implements` keyword (e.g., `Comparable<MyClass>`). This interface has a single `.compareTo()` method that returns an `int` value based on whether the `value` of a current class instance (referenced by `this`) can be logically sorted with the value of another instance of the same class.
+
+| Return Value | Meaning                         |
+| :----------: | ------------------------------- |
+|    `>= 1`    | this instance > passed instance |
+|     `0`      | this instance = passed instance |
+|   `<= -1`    | this instance < passed instance |
```

Comment:
```suggestion
|    >= 1    | this instance > passed instance |
|     0      | this instance = passed instance |
|   <= -1    | this instance < passed instance |
```

Although this is return output, I'm not sure if they should necessarily be in back-ticks (to [a point made in a separate PR](https://github.com/Codecademy/docs/pull/840#discussion_r919231406)).